src/wrapped/wrappedsdl2.c

```diff
@@ -19,6 +20,8 @@
 #include "myalign.h"
 #include "threads.h"
 
+#include "wrappedsdl2defs.h"
```

Comment:
Isn't it
```C
#include "generated/wrappedsdl2defs.h
```
instead?

Reply:
Got auto-completed by the IDE, assumed that was correct since it worked.